HISTORY.md

```diff
@@ -2,12 +2,22 @@
 
 http://visjs.org
 
-## 2019-07-19, version 6.0.0
+## 2019-10-21, version 6.0.0
 
 In version 6.0.0, [timeline-plus](https://github.com/yotamberk/timeline-plus) features and PRs were ported into this library.
+We are now connected to circle-ci for future version publishes.
 
 ### New Features
 
+- feat: add an option to make the custom marker editable ([vis-timeline#117](https://github.com/visjs/vis-timeline/pull/117)) - thanks to @ryamaguchi0220
```

Comment:
for issues within the same project it is enough to write #117 without a link or anything.